src/main/resources/assets/additionalguns/models/item/m16a2.json

```
@@ -1,5 +1,5 @@
 {
-	"credit": "Made with Blockbench. Made by VIKTORWOOD",
+	"credit": "Made with Blockbench by VIKTORWOOD. Modified by Igggosha",
```

Looks good! But I would suggest you change this line back to its original state because we always try to credit the original author of the model:
```suggestion
	"credit": "Made with Blockbench. Made by VIKTORWOOD",
```